javascript/javascript2/week3/giphyApi/style.css

```diff
@@ -0,0 +1,45 @@
+main {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    background: azure;
+}
+h1 {
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS',
+        sans-serif;
+    font-style: oblique;
+    font-size: 5rem;
+    color: aqua;
+    padding: 0;
+    margin: 0;
+}
+main * {
+    margin-bottom: 1em;
+}
+input[type='text'] {
+    border: none;
+    outline: none;
+    border-radius: 24px;
+    padding: 1em;
+    background: #0a0a0a;
+    color: white;
+    font-size: 1rem;
+}
+#search-gif-button {
```

Comment:
Always try skipping id, rather use class, unless you must use it, though that is not a common use case.